runtime/rococo/src/validator_manager.rs

```diff
@@ -67,7 +67,7 @@ pub mod pallet {
 		///
 		/// The new validators will be active from current session + 2.
 		#[pallet::call_index(0)]
-		#[pallet::weight(100_000)]
+		#[pallet::weight({100_000})]
```

Comment:
This will get a deprecation warning in the future anyway when we deprecate the `From<u64> for Weight` hacks.